### PR TITLE
Add Aspect BSD to LBtoL0 and test data.

### PR DESCRIPTION
**Pull request by samaloney**

### CODE REVIEW

File: stixcore/processing/LBtoL0.py [hunk 1]
```diff
@@ -95,7 +50,14 @@ def process_fits_files(self):
                         service_subtype=cur_lb.service_subtype, ssid=cur_lb.ssid, data=None,
                         control=None)
 
-                    level0 = tmp.from_levelb(comp)
+                    # TODO need to carry better information for logging like index from oginianl
+                    # files and file names
```
Comment (nicHoch) [inline stixcore/processing/LBtoL0.py:54]: i created: https://github.com/i4Ds/STIXCore/issues/85

File: stixcore/products/level0/science.py [hunk 2]
```diff
@@ -277,3 +280,96 @@ def __init__(self, control, data):
...
+        # Data
+        try:
+            data = Data()
+            data['time'] = time
+            data['timedel'] = timedel
+            data['cha_diode0'] = packets.get_value('NIX00090')
+            data['cha_diode1'] = packets.get_value('NIX00091')
+            data['chb_diode0'] = packets.get_value('NIX00092')
+            data['chb_diode1'] = packets.get_value('NIX00093')
+            data['control_index'] = np.hstack([np.full(ns, i) for i, ns in enumerate(samples)])
```
Comment (nicHoch) [inline stixcore/products/level0/science.py:343]: should we set the .meta ?

**Review by nicHoch — APPROVED** [review]
only the minor meta comment

File: stixcore/products/level0/science.py (hunk 2 above)
Comment (samaloney) [reply stixcore/products/level0/science.py:343]: Yea good spot I will update

**Comment by samaloney** [comment]
Need to look at #84

File: stixcore/products/level0/science.py (hunk 2 above)
Comment (samaloney) [reply stixcore/products/level0/science.py]: I've just added them manually I'm sure there is cleaner way will comeback once we have the other BSD merged.

**Pull request merged**